lib/plugins/aws/deploy/compile/events/apiGateway/lib/method/integration.js

```diff
@@ -174,6 +179,11 @@ module.exports = {
       "principalId": "$context.authorizer.principalId",
       "stage": "$context.stage",
 
+      "cognitoPoolClaims" : {
```

Comment:
They are the same, however there are currently two templates, one for JSON and one for WWW forms. I'm about commit something that refactors this.